packages/nextjs/src/server/keyless-node.ts

```diff
@@ -46,7 +53,8 @@ function updateGitignore() {
   const { existsSync, writeFileSync, readFileSync, appendFileSync } = safeNodeRuntimeFs();
```

Comment:
Shouldn't this logic be behind a guard clause that checks for existence of the node APIs? Then we don't need to do spread this safeNode** into multiple functions